src/app/feed/feed-card/feed-card.component.ts

```diff
@@ -10,11 +11,13 @@ import { Observable } from 'rxjs';
 export class FeedCardComponent implements OnInit {
   @Input() data: any;
   imageUrl: Observable<string>;
+  comments: Observable<any[]>;
```

Comment:
Maybe strong type to a string?

Reply:
I don't know if changing the type would affect the logic of Firebase's API. If we're worried about emojis, we shouldn't be: they are defined as [UniCode](https://guigarage.com/2015/01/support-emojis/)

Reply:
Again, this is not a string, this is an array of objects which we should create a class to represent, but I haven't because we were likely to change it in the database.